nginx/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# a minimal Nginx container including Containerbuddy and a simple virtulhost config
+FROM gliderlabs/alpine:3.3
```

Comment:
We go back and forth on whether Nginx is a reusable image or one that should be built for each project.

https://github.com/tgross/triton-nginx supports [fetching the Nginx config from Consul or injecting it via an environment var](https://github.com/tgross/triton-nginx/blob/master/bin/reload.sh#L26-L35). The goal there is to make it more reusable in that way.

https://www.joyent.com/blog/applications-on-autopilot#nginx and some other examples demonstrate building Nginx with the rest of the app so we can more clearly highlight how to use Containerbuddy, Consul, and consul-template.